bundles/ranvier-areas/areas/limbo/items.yml

```diff
@@ -10,6 +10,9 @@
   roomDesc: "A moldy slice of cheese"
   keywords: [ "slice", "cheese", "moldy" ]
   description: "A yellow, slightly moldy slice of cheese. Only a rat could find this appetizing."
+  behaviors:
```

Comment:
Weird (though, it makes sense). 

So, maybe there should be a decay event as well, or should force the item to be dropped by emitting the drop event?